### ISSUE TITLE
Manual Install - Node Failed to Load, Import Failed

### ISSUE THREAD

**Issue by brbbbq** [yuvraj108c/ComfyUI-Upscaler-Tensorrt#24]
I tried following your advice from the [previous issue](https://github.com/yuvraj108c/ComfyUI-Upscaler-Tensorrt/issues/22), but I can't even get the node to load at this point. I did a full deletion of the custom node folder and local pip packages, manually downloaded the node files from git, and ran pip install, but when I open ComfyUI, the node fails to load. When I check the Manager, it says "IMPORT FAILED". I tried the "Try Fix" button and reset, but no luck.
![240916-125009-chrome](https://github.com/user-attachments/assets/7cf7d878-bf3a-4005-9c44-4b0685bd094c)
![240916-124754-chrome](https://github.com/user-attachments/assets/6a9a5477-c4be-4937-acd6-876df1542301)

```
ComfyUI: 2652[0c7c98](2024-09-05)
Manager: V2.50.3
```

These are all the steps I followed:

Ran pip uninstall tensorrt:
```
>pip uninstall tensorrt
```

Deleted custom node folder, and "tensor" folders from site-packages:
```
J:\ComfyUI_windows_portable-3\python_embeded\Lib\site-packages\tensorrt
J:\ComfyUI_windows_portable-3\python_embeded\Lib\site-packages\tensorrt_cu12-10.4.0.dist-info
J:\ComfyUI_windows_portable-3\python_embeded\Lib\site-packages\tensorrt-10.4.0.dist-info
```

Cloned the github library:
```
>git clone https://github.com/yuvraj108c/ComfyUI-Upscaler-Tensorrt
Cloning into 'ComfyUI-Upscaler-Tensorrt'...
remote: Enumerating objects: 67, done.
remote: Counting objects: 100% (67/67), done.
remote: Compressing objects: 100% (53/53), done.
remote: Total 67 (delta 35), reused 25 (delta 11), pack-reused 0 (from 0)
Receiving objects: 100% (67/67), 51.04 KiB | 2.43 MiB/s, done.
Resolving deltas: 100% (35/35), done.
```

Pip installed tensorrt:
```
>pip install tensorrt
Collecting tensorrt
  Using cached tensorrt-10.4.0.tar.gz (16 kB)
  Preparing metadata (setup.py) ... done
Collecting tensorrt-cu12==10.4.0 (from tensorrt)
  Using cached tensorrt-cu12-10.4.0.tar.gz (18 kB)
  Preparing metadata (setup.py) ... done
Building wheels for collected packages: tensorrt, tensorrt-cu12
  Building wheel for tensorrt (setup.py) ... done
  Created wheel for tensorrt: filename=tensorrt-10.4.0-py2.py3-none-any.whl size=16353 sha256=99beca4568ea290a3d7424902c0c4b35ce16465609e3cc03ca18db42f6e7e889
  Stored in directory: c:\users\...\appdata\local\pip\cache\wheels\a0\19\5f\9e37c904409cb83caec8e2b682732c4cc7a74adbeda520edab
  Building wheel for tensorrt-cu12 (setup.py) ... done
  Created wheel for tensorrt-cu12: filename=tensorrt_cu12-10.4.0-py2.py3-none-any.whl size=17571 sha256=dfba58d659b0fa4e7c48b3182d549eabf216e6009447f4b615541f3cc3725fd3
  Stored in directory: c:\users\...\appdata\local\pip\cache\wheels\76\26\d6\92ecf38513ce11b54032131c94c117db6f1b35752d00728b29
Successfully built tensorrt tensorrt-cu12
WARNING: Error parsing dependencies of pytorch-lightning: .* suffix can only be used with `==` or `!=` operators
    torch (>=1.9.*)
           ~~~~~~^
WARNING: Error parsing dependencies of tensorrt-cu12-bindings: [Errno 2] No such file or directory: 'c:\\users\\...\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\tensorrt_cu12_bindings-10.0.1.dist-info\\METADATA'
WARNING: Error parsing dependencies of tensorrt-cu12-libs: [Errno 2] No such file or directory: 'c:\\users\\...\\appdata\\local\\programs\\python\\python310\\lib\\site-packages\\tensorrt_cu12_libs-10.0.1.dist-info\\METADATA'
Installing collected packages: tensorrt-cu12, tensorrt
  Attempting uninstall: tensorrt-cu12
    Found existing installation: tensorrt-cu12 10.0.1
    Uninstalling tensorrt-cu12-10.0.1:
      Successfully uninstalled tensorrt-cu12-10.0.1
Successfully installed tensorrt-10.4.0 tensorrt-cu12-10.4.0
```

I was able to re-exported the engine, but of course I can't test it with the node failing to load. Sorry to complicate the issue!

**Comment by brbbbq** [comment]
Oh wow. You know what? I got it to work. I noticed there were more site-packages for the old installation than for the new installation. I went ahead and deleted everything from the new installation, then copied the site-packages, the custom nodes folder, and the engines from the old installation to the new installation, and it worked!

Site-packages folders:
![240916-173054-explorer](https://github.com/user-attachments/assets/4203b980-0145-4831-84bd-61fe2652ad3e)

Custom nodes folder:
![240916-172854-explorer](https://github.com/user-attachments/assets/bad57bda-2bac-4bac-a3ed-cdc29e94d627)

Engines:
![240916-172946-explorer](https://github.com/user-attachments/assets/7c1af9a1-86a8-4901-a78a-9e3fe21c6846)

Here's the difference between the site-packages folders between the 2 installations:
![240916-172439-explorer-tensorrt_site-packages](https://github.com/user-attachments/assets/b7a76d77-9adb-45b0-bbb5-4f041deec416)

So I was able to get it to work on my system, but I'm assuming there's still something wrong with the current installation method that's preventing it from working new.
